Find a functionally equivalent code:android.ddm.DdmHandleHeap.handleHPGC:COMMENT
Method Modifier: private     
Comment:/*
     * Handle a "HeaP Garbage Collection" request.
     */

@@ -1,7 +1,7 @@
 {
     if (false)
         Log.d("ddm-heap", "Heap GC request");
-    System.gc();
+    Runtime.getRuntime().gc();
     // empty response
     return null;
 }

